instrumentation/quarkus-resteasy-reactive/quarkus2-testing/build.gradle.kts

```diff
@@ -10,6 +10,7 @@ otelJava {
 
 dependencies {
   implementation(enforcedPlatform("io.quarkus.platform:quarkus-bom:2.16.7.Final"))
+  implementation(enforcedPlatform("org.junit:junit-bom:5.10.3"))
```

Comment:
can you add a comment if this is intentional not to use 5.11.0?

Reply:
Added a comment. I think quarkus also sets junit version and we end up with jars with conflicting versions. This PR just sets the last version where things worked.